azurerm/internal/services/desktopvirtualization/virtual_desktop_host_pool_resource.go

```diff
@@ -99,6 +99,11 @@ func resourceVirtualDesktopHostPool() *schema.Resource {
 				Default:  false,
 			},
 
+			"custom_rdp_property": {
```

Comment:
Could we use 
```suggestion
			"custom_rdp_properties": {
```
as property implies 1, and thd documentation here suggests it actually sends multiple (there might also be an argument to make it a typelist too but thats your call)

Reply:
Not even Microsoft is bothered to do any kind of API for every separate setting, they just have a javascript that translates checkboxes to rdp properties string. So in short, this is good enough I think. :) 

Agreed on schema, was mostly following API naming.